hack/get-operator-sdk.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Download latest release of operator-sdk, intended for use by ../Makefile
+
+set -e
+BASE=https://github.com/operator-framework/operator-sdk
+VERSION="$(curl -fLsS -o /dev/null -w '%{url_effective}' $BASE/releases/latest | sed 's-.*/--')"
+URL=$BASE/releases/download/$VERSION/operator-sdk-$VERSION-$(uname -i)-linux-gnu
+echo downloading $URL
+curl -fL -o bin/operator-sdk $URL && chmod +x bin/operator-sdk
+bin/operator-sdk version > /dev/null # Verify executable works
```

Comment:
shouldn't this be `$GOPATH/bin/operator-sdk` ? thats where i currently have my operator-sdk binary...

Reply:
The makefile will pick it up from PATH or download a released binary into bin/ - way back when I started using it to generate CRDs - this tool doesn't work with normal `go get` and I ran into issues building it from master. We should revisit this, when we update our operator-sdk library dependencies to a modern version, we can just build or  `go run` the same version of the tool as we are using for the libraries and avoid all the downloading muck.

Reply:
sorry, i realize i wrote my comment on the execution line, but this applies to where we pull it down from `curl` `-o bin/operator-sdk`

Reply:
Yep, currently we're caching those binaries in the local bin/ directory. I don't want to put something in GOPATH/bin that wasn't locally built  from GOPATH. We should revisit this once we move to `go mod`  which has a much better story for locally caching multiple versions of libs & binaries without trampling on the users
 GOPATH.